src/acvp_transport.c

```diff
@@ -775,6 +775,26 @@ ACVP_RESULT acvp_get_request_status (ACVP_CTX *ctx,
 }
 
 
+ACVP_RESULT acvp_post(ACVP_CTX *ctx, 
```

Comment:
Don;t need new function once you get the string length as mentioned above

Reply:
done